tango_simlib/tests/test_fandango_json_parser.py

```diff
@@ -0,0 +1,196 @@
+"""This module tests the fandango_json_parser script.
+"""
+import unittest
+import logging
+import pkg_resources
+
+import tango
+
+from tango_simlib.utilities import fandango_json_parser
+
+MODULE_LOGGER = logging.getLogger(__name__)
+
+
+EXPECTED_DEVICE_PARAMETERS = [
+    'commands', 'class_properties', 'name',
+    'level', 'server', 'host', 'dev_class',
+    'attributes', 'properties'
+    ]
+
+EXPECTED_CMD_PROPERTY_PARAMETERS = [
+    'name', 'dtype_in',
+    'doc_in', 'dtype_out',
+    'doc_out'
+    ]
+
+INTERESTED_CMD_PROP = [
```

Comment:
This is a list, so suggest the name ends in "S", for the plural.  Same for `INTERESTED_ATTR_PROP`